Samples.Swift/Samples.Swift/ViewControllers/CheckoutViewController/Views/ItemCell.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ItemCell.swift
+//  paypalcheckout-iOS-swift-sample
+//
+//  Created by Jax DesMarais-Leder on 3/10/21.
+//
+
+import UIKit
+import PayPalCheckout
+
+class ItemCell: UITableViewCell {
```

Comment:
I notice this cell and `AddItem` cells remain highlighted after I close `AddItemViewController`.

If you want to have it highlighted, we should handle deselecting the cells, something like this:
```
override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
    tableView.deselectRow(at: indexPath, animated: true)    // this will remove highlight
}
```

Or maybe we can make these cells' `selectionStyle = .none` for simplicity. It's up to you :smile: